validators/email-question.validator.js

```diff
@@ -0,0 +1,32 @@
+const Joi = require('joi');
+const { emailRegExp } = require('../consts/regexp');
+
+const questionInputValidator = Joi.object({
+  senderName: Joi.string()
+    .min(2)
+    .max(30)
+    .trim()
+    .required(),
+  text: Joi.string()
+    .min(2)
+    .max(700)
```

Comment:
Please remove .max validation, it could be a loooong question

Reply:
fixed